tests/python/contrib/test_hexagon/test_vtcm_bandwidth.py

```diff
@@ -104,8 +106,8 @@ def evaluate(hexagon_session, sch, size):
     )
 
     # These are reduced for CI but number=100 and repeat=10 does a good job of removing noise.
-    number = 1
-    repeat = 1
+    number = 10
+    repeat = 10
```

Comment:
Did you mean to change this?  Makes the test run longer in CI.

Reply:
Same as above, should have added a check for CI awhile ago.